### DISCUSSION TITLE
Possibility to open and list content of volume from longhorn ui

### DISCUSSION THREAD

**Discussion by w3blogfr** [longhorn/longhorn#6809] [Ideas]
Hello,
It would be very nice to have the possibility to view a volume content from Longhorn UI and to have possibility to export the content.

## Having a maintenance container to see volume content

I have a strimzi pod that continuously restart due to error on the disk. This pod is managed by an operator, I don't have the possibility to delete the pods without having having to stop the Strimzi operator. As volume is read-write single node, I can't run a new ubuntu image and mount the volume there to debug it.
It would be wonderfull to have the possibility from longhorn UI to open a volume in maintenance mode (blocking attachment to any pods), and run a linux container to be able to see the volume content. Something similar to rancher when we want to execute Kubectl.

Then we close it and stop the maintenance, and the volume could be attached again to normal containers.

## Export volume

Another nice feature would be to have possibility to export a volume from the UI, with a zip or something similar. Actually, backup are not readable, and require lot of effort to get back the volume, mount the volume somewhere (with CSI) just to visualize the content.

Regards.

**Comment by c3y1huang** [comment]
Longhorn primarily operates at the block storage level, rather than the filesystem level. This means it lacks awareness of the actual data contents within the volume. Therefore, to view or export content directly appears to be a feature request that is not in line with its current capabilities.

cc @innobead , could you please review this to see if we should convert this into a feature request?

**Comment by mistuttodos** [comment]
Any news on this?
